distribution/ddf-common/src/main/resources-filtered/etc/system.properties

```diff
@@ -24,6 +24,11 @@
 
 #START DDF SETTINGS
 
+#
+# System branding
+#
+org.codice.ddf.system.branding=${branding-lowercase}
```

Comment:
Could this property name change to not include "ddf" so that it can be reused in downstream projects?